docs/contributing/minutes.md

```diff
@@ -4,6 +4,73 @@ title: Team Minutes
 sidebar_label: Team Minutes
 ---
 
+## 03-02-2022
+
+Attendees: Ali, Greg, Sally, Daniel, Mitch
+
+### Agenda
+
+- Celebrating fundamentals
+- Welcoming Sanyia
+- Sprint planning
+- Plan for 2022
+
+### Celebrating fundamentals
+
+- Woop! 🎉 Go Fundamentals - now published!
+- Fundamentals training starting on 12th Match for S African volunteers
+- Fundamentals starts in April with South Africa Class 2
+
+### Plan 2022
+
+- Improve student tracking
+- Further delegation to module teams
```

Comment:
@gregdyke Think you went through a list of post-its but I may have missed off one or 2 if you want add them in here